integration_tests/test_os.py

```diff
@@ -0,0 +1,14 @@
+from ltypes import i64
+from os import (open, read, close)
+
+def test():
+    path: str 
+    path = "integration_tests/test_os.py"
+    fd: i64
+    n: i64
+    fd = open(path)
```

Comment:
Here, Python takes a minimum of two arguments ie.,
Syntax:
```python
os.open(path, flags[, mode])
```
Example:
```python
os.open(path, O_RDONLY)
```
The problem is how do I pass the constant like `O_RDONLY`, `O_WRONLY` or bitwise OR operation like `O_RDWR | O_CREAT` to C
cc @certik @Smit-create @namannimmo10

Reply:
> The problem is how do I pass the constant like O_RDONLY, O_WRONLY or bitwise OR operation like O_RDWR | O_CREAT to C

I think we have to implement bit wise operations in LPython and then pass the flags as an integer (to C). Go ahead and open up an issue for this and we'll work it.